src/es/hentaimode/src/eu/kanade/tachiyomi/extension/es/hentaimode/HentaiMode.kt

```diff
@@ -0,0 +1,164 @@
+        throw UnsupportedOperationException()
+    }
+
+    override fun latestUpdatesFromElement(element: Element): SManga {
+        throw UnsupportedOperationException()
+    }
+
+    override fun latestUpdatesNextPageSelector(): String? {
+        throw UnsupportedOperationException()
+    }
+
+    // =============================== Search ===============================
+    override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> {
+        return if (query.startsWith(PREFIX_SEARCH)) { // URL intent handler
+            val id = query.removePrefix(PREFIX_SEARCH)
+            client.newCall(GET("$baseUrl/g/$id"))
+                .asObservableSuccess()
+                .map(::searchMangaByIdParse)
+        } else {
+            super.fetchSearchManga(page, query, filters)
+        }
+    }
+
+    private fun searchMangaByIdParse(response: Response): MangasPage {
+        val doc = response.asJsoup()
+        val details = mangaDetailsParse(doc)
+            .apply { setUrlWithoutDomain(doc.location()) }
+        return MangasPage(listOf(details), false)
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        require(query.length >= 3) { "Please use at least 3 characters!" }
+        return GET("$baseUrl/buscar?s=$query")
+    }
+
+    override fun searchMangaSelector() = popularMangaSelector()
+
+    override fun searchMangaFromElement(element: Element) = popularMangaFromElement(element)
+
+    override fun searchMangaNextPageSelector() = null
+
+    // =========================== Manga Details ============================
+    private val additionalInfos = listOf("Serie", "Tipo", "Personajes", "Idioma")
+
+    override fun mangaDetailsParse(document: Document) = SManga.create().apply {
+        thumbnail_url = document.selectFirst("div#cover img")?.absUrl("src")
+        status = SManga.COMPLETED
+        update_strategy = UpdateStrategy.ONLY_FETCH_ONCE
+        with(document.selectFirst("div#info-block > div#info")!!) {
+            title = selectFirst("h1")!!.text()
+            genre = getInfo("Categorías")
+            author = getInfo("Grupo")
+            artist = getInfo("Artista")
+
+            description = buildString {
+                additionalInfos.forEach { info ->
+                    getInfo(info)?.also { append("$info: $it\n") }
+                }
+            }
+        }
+    }
+
+    private fun Element.getInfo(text: String): String? {
+        return select("div.tag-container:containsOwn($text) a.tag")
+            .joinToString { it.text() }
+            .takeUnless(String::isBlank)
+    }
+
+    // ============================== Chapters ==============================
+    override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
+        val chapter = SChapter.create().apply {
+            url = manga.url.replace("/g/", "/leer/")
+            chapter_number = 1F
+            name = "Hentai"
```

Comment:
Placeholder chapters are usually named "Chapter". Not something to strictly follow though.